### PR TITLE
add healthcheck doc

### PR DESCRIPTION
**Pull request by zhengyangyong**
Signed-off-by: zhengyangyong <yangyong.zheng@huawei.com>

### CODE REVIEW

File: _users/healthcheck.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: "HealthCheck"
+lang: cn
```
Comment (WillemJiang) [inline _users/healthcheck.md]: language should be en.

File: _users/healthcheck.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ref: healthcheck
+permalink: /users/healthcheck/
+excerpt: "健康检查"
```
Comment (WillemJiang) [inline _users/healthcheck.md]: this part should be english.

File: _users/healthcheck.md (hunk 1 above)
Comment (zhengyangyong) [reply _users/healthcheck.md]: oh...sorry

File: _users/healthcheck.md (hunk 2 above)
Comment (zhengyangyong) [reply _users/healthcheck.md]: oh...sorry

File: _users/healthcheck.md (hunk 1 above)
Comment (zhengyangyong) [reply _users/healthcheck.md]: Fixed

File: _users/healthcheck.md (hunk 2 above)
Comment (zhengyangyong) [reply _users/healthcheck.md]: Fixed

File: _users/cn/healthcheck.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  @Override
+  public HealthCheckResult check() {
+    //add your health check code here
+    Connection connection = null;
+    try {
+      connection = DriverManager.getConnection("jdbc:mysql://localhost:3306/test_db?useSSL=false", "root", "pwd");
+      return new HealthCheckResult(true, "local mysql health check", "");
+    } catch (SQLException e) {
+      e.printStackTrace();
```
Comment (WillemJiang) [inline _users/cn/healthcheck.md]: It's not a good way to printStackTrace out directly.

**Comment by WillemJiang** [comment]
Just changed e.printStackTrace() to LOG.error(xxx).

**Pull request merged**